tests/unit_tests/test_obstacle_map.py

```diff
@@ -358,3 +358,13 @@ def test_get_path_distance(self):
 		self.paths1 = self.obstacle_map1.generate_possible_paths(self.test_obstacle1)
 		self.min_path1 = self.obstacle_map1.get_min_path(self.paths1)
 		self.assertEqual(self.obstacle_map1.get_path_distance(self.min_path1), 236.44394938110665)
+
+	def test_is_moving_obstacle_in_path(self):
```

Comment:
We do not use the current SDA system for moving obstacles anymore. Please update your comment accordingly.